### PR TITLE
create logout feature

### PR DESCRIPTION
**Pull request by Maheima**
Closes issue number:  https://github.com/Real-Dev-Squad/website-status/issues/346

### Title:
Added logout button on the status site.

### Description:
Add a dropdown with user profile and signout link in it

### CODE REVIEW

**Review by ivinayakg — COMMENTED** [review]
LGTM. 
great work Mahima 👍.

**Review by ivinayakg — DISMISSED** [review]
LGTM 👍

**Review by YashJain24-chief — DISMISSED** [review]
Looks good to me, great work Mahima

File: src/components/navBar/navBar.module.scss [hunk 1]
```diff
@@ -39,38 +39,45 @@ $theme-active: #87d870;
...
+  }
+  .signInLink {
+    border-radius: 6px;
+    padding: 5px;
+    background-color: $theme-blue;
+    color: $white;
+    border: 1px solid $white;
+    display: flex;
+    justify-content: space-around;
+    font-size: 13px;
```
Comment (harshith-venkatesh) [inline src/components/navBar/navBar.module.scss:51]: Why are we not using rem here

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import Link from 'next/link'
+import styles from '@/components/Dropdown/dropdown.module.scss';
+import { USER_PROFILE_URL } from '../constants/url';
+
+
+const Dropdown = () => {
+
+  const logout = () => {
+    fetch('https://api.realdevsquad.com/auth/signout', {
```
[the remark this answers is not in the document]
Comment (Maheima) [reply src/components/Dropdown/Dropdown.tsx:9]: Yes, I'll do that

File: src/components/navBar/navBar.module.scss (hunk 1 above)
Comment (harshith-venkatesh) [reply src/components/navBar/navBar.module.scss:51]: You can reach out to AO of the site for the above decision 😁
Well, you can the pros and cons going ahead with rem

Comment (Maheima) [reply src/components/navBar/navBar.module.scss:51]: I am the AO and I was thinking of probably creating a ticket for it rather.

**Review by yesyash — APPROVED** [review]

**Review by harshith-venkatesh — APPROVED** [review]

File: src/components/navBar/navBar.module.scss (hunk 1 above)
Comment (Maheima) [reply src/components/navBar/navBar.module.scss:51]: done

**Pull request merged**